pkg/sqlqueries/SQL/mto/milestone/create_allow_conflicts.sql

```diff
@@ -0,0 +1,74 @@
+WITH retVal AS (
+    INSERT INTO mto_milestone(
+        id,
+        model_plan_id,
+        mto_common_milestone_key,
+        mto_category_id,
+        name,
+        facilitated_by,
+        facilitated_by_other,
+        need_by,
+        status,
+        risk_indicator,
+        is_draft,
+        created_by,
+        created_dts
+    )
+    VALUES (
+        :id,
+        :model_plan_id,
+        :mto_common_milestone_key,
+        :mto_category_id,
+        :name,
+        :facilitated_by,
+        :facilitated_by_other,
+        :need_by,
+        :status,
+        :risk_indicator,
+        :is_draft,
+        :created_by,
+        COALESCE(:created_dts, CURRENT_TIMESTAMP)
+    )
+    ON CONFLICT (model_plan_id, mto_common_milestone_key) 
+    DO UPDATE SET
```

Comment:
I don't think we should touch this if it already exists, I think it might be a problematic paradigm for change history. I think we currently do nothing in these cases, and only update when we explicitly update.